vhdbuilder/packer/list-images.sh

```diff
@@ -3,9 +3,31 @@ set -euxo pipefail
 
 CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
 
+function generate_image_bom_for_containerd() {
+    temp_image_bom=/opt/azure/containers/temp-image-bom.json
+    IFS_backup=$IFS
+    IFS=$'\n'
+    ctr_list=$(ctr --namespace k8s.io image list | sed 1d | awk '{print $1, $3}')
+    crictl_list=$(crictl images --no-trunc | sed 1d | awk '{printf "%s:%s %s\n", $1,$2,$3;}')
+
+    for image in $ctr_list; do
+        tag=$(echo $image | awk '{print $1}')
+        # skip image tags that consist of a sha256 hash
```

Comment:
i vaguely recall this. you get one line per tag/sha pair per image, or something? why is this necessary? we should only have tagged images so IIRC this was something funny with ctr behavior

Reply:
yep, each line is "<tag> <sha>" from the ctr output, then each line is "<tag> <id>" from the crictl output

Reply:
and yeah not exactly sure why, but ctr will output a bunch of tags that are just shas in and of themselves, I cross-referenced previous image-bom's and saw that these images aren't included, so I added this to simply skip them for now